pages/product/index.vue

```diff
@@ -19,79 +49,113 @@
             class="product__drag-icon smartphone-only"
           />
         </div>
+
         <div class="product__price-and-rating">
           <SfPrice
             :regular="$n(productGetters.getPrice(product), 'currency')"
             :special="
               product.price.salePrice && $n(productGetters.getSalePrice(product), 'currency')
             "
           />
-          <div>
-            <div class="product__rating">
-              <SfRating :score="0" :max="5" />
-              <a v-if="!!product.totalReviews" href="#" class="product__count">
-                ({{ product.totalReviews }})
-              </a>
-            </div>
+
+          <div class="product__rating">
+            <SfRating :score="3" :max="5" />
+            <a v-if="!!product.totalReviews" href="#" class="product__count">
+              ({{ product.totalReviews }})
+            </a>
+
             <SfButton class="sf-button--text" data-testid="read-all-reviews" @click="changeTab(2)">
-              Read all reviews
+              Write a Review
```

Comment:
Add an entry to the "lang/en.js" for "Write Review" and use here with the i18n global $t("Write Review")

Reply:
Resolved